### PR TITLE
Fill in the Reg C gaps

### PR DESCRIPTION
**Pull request by grapesmoker**
This PR adds the Reg C notices that were missing from the initial build of Reg C because we didn't have the diff capability yet. The notice 2015-26607 contains three separate effective dates, and those are broken out into three separate notice files, with the appropriate dependencies on prior versions. The regulation versions compiled with those notices are also added.

### CODE REVIEW

**Comment by chosak** [comment]
@grapesmoker would you mind resolving these conflicts before I review this?

**Comment by grapesmoker** [comment]
I'm trying but I'm not sure what they are. I was able to merge master into the branch without any problems...

**Comment by grapesmoker** [comment]
The problem is that it won't show me what the conflicts are; when I try to merge master into the branch, I don't actually see any conflicts there.

**Comment by chosak** [comment]
I can reproduce the conflicts locally:

```sh
$ git fetch
$ git checkout reg-c-gaps
$ git merge origin/master
```

**Comment by grapesmoker** [comment]
Last time I did that it didn't work, but now it does. [shakes fist] GIIITTTTTT! I'll fix and try again.

**Comment by grapesmoker** [comment]
Ok, I think I have fixed all the conflicts.

File: notice/1003/2015-26607_20200101.xml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<notice xmlns="eregs" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" schemaLocation="eregs https://raw.githubusercontent.com/cfpb/regulations-schema/master/src/eregs.xsd">
```
Comment (chosak) [inline notice/1003/2015-26607_20200101.xml]: This line fails RegML schema validation, it should be `xsi:schemaLocation` instead of `schemaLocation`.

**Review by chosak — CHANGES_REQUESTED** [review]

**Comment by grapesmoker** [comment]
oops

**Review by chosak — APPROVED** [review]
Thanks; these content changes are far too complex to go through line-by-line but I don't see anything major jumping out at me in the structure of the changes.

**Comment by grapesmoker** [comment]
Yeah, there's nothing really major here. It's just the same content that was in the regulation files but broken out into notices.

**Pull request merged**